pkg/globalmanager/runtime/worker.go

```diff
@@ -169,3 +175,29 @@ func createEnvVars(envMap map[string]string) []v1.EnvVar {
 	}
 	return envVars
 }
+
+// injectModelHotUpdateMount injects volume mounts when worker supports hot update of model
+func injectModelHotUpdateMount(pod *v1.Pod) {
+	hostPathType := v1.HostPathDirectoryOrCreate
+
+	var volumes []v1.Volume
+	var volumeMounts []v1.VolumeMount
+
+	volumeName := ConvertK8SValidName(ModelHotUpdateVolumeName)
+	volumes = append(volumes, v1.Volume{
+		Name: volumeName,
+		VolumeSource: v1.VolumeSource{
+			HostPath: &v1.HostPathVolumeSource{
+				Path: ModelHotUpdateHostPrefix,
```

Comment:
only mount pod relative path, ModelHotUpdateHostPrefix contains all pods' information

Reply:
fix it